packages/opencensus-nodejs/README.md

```diff
@@ -44,22 +44,7 @@ tracing.start();
 
 ### Manually Instrument an Application
 
-In addition to automatic tracing, it is possible to manually create your own root and child spans.
-
-```typescript
-const rootSpanOptions = { name: 'your root span' };
-tracing.tracer.startRootSpan(rootSpanOptions, (rootSpan) => {
-
-    // You can create as many child spans as needed
-    childSpan = tracing.tracer.startChildSpan({ name: 'name of your child span' });
-    // Do some operation...
-    // Finish the child span at the end of it's operation
-    childSpan.end();
-
-    // Finish the root span at the end of the operation
-    rootSpan.end();
-});
-```
+For manual insturmentation see the `@opencensus/nodejs-base` package.
```

Comment:
What if someone wants both automatic and manual instrumentation? I would think they would be able to do it by using the `@opencensus/nodejs` package as well? Maybe we should still have the doc above, but mention that if you *only* want manual instrumentation to see the `@opencensus/nodejs-base` package?

Reply:
That's a very good point, I'll revert it and add the extra context.